frontend/src/models/fms_api_models.ts

```diff
@@ -105,13 +105,18 @@ export interface FMSArchivedComment extends FMSTrackedModel {
     comment: string
 }
 
+export enum ReleaseStatus {
+    AVAILABLE = 0,
+    RELEASED = 1,
+    BLOCKED = 2,
+}
 export interface FMSReadset extends FMSTrackedModel {
     id: FMSId                          // Unique ID of object in database
     name: string                       // External name that identifies the readset if the run did not come from Freezeman
     sample_name: string                // Name that identifies the sample if the run did not come from Freezeman
     derived_sample: FMSId              // Derived sample matching the readset
     sample_source: FMSId               // Last non pool sample (if any, else last pool) before experiment
-    release_status: number              // The file's release status (AVAILABLE = 0, RELEASED = 1,BLOCKED = 2)
+    release_status: ReleaseStatus      // The file's release status (AVAILABLE = 0, RELEASED = 1,BLOCKED = 2)
```

Comment:
i think the comment is not longer needed ;).

Reply:
`validation_status` is next